staging/src/k8s.io/kubectl/pkg/cmd/cp/cp.go

```diff
@@ -304,7 +304,7 @@ func (o *CopyOptions) copyFromPod(src, dest fileSpec) error {
 		cmdutil.CheckErr(err)
 	}()
 	prefix := getPrefix(src.File)
-	prefix = path.Clean(prefix)
+	prefix = path.Dir(path.Clean(prefix))
```

Comment:
Actually, this is wrong, I need to change that prefix in the place where the `destFileName` is constructed.